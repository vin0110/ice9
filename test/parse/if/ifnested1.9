if true ->
  if 8 -> 1; fi
fi

