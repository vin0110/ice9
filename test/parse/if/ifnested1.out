IF:
  BLIT: TRUE
(then)
  IF:
    ILIT:8
  (then)
    ILIT:1
  (else)
    nil
(else)
  nil
